extensions/twitch/src/index.tsx

```diff
@@ -47,14 +44,11 @@ export default function main() {
           <ActionPanel>
```

Comment:
This action appears almost everywhere! You can create utils.tsx and link them here